lang/pypy/Portfile

```
@@ -102,32 +129,41 @@ subport pypy39 {
     select.entries-append [list python3 python3-$subport $subport]
 }
 
-
 subport pypy-tkinter {
-    set python.branch   2.7
-}
-
-subport pypy36-tkinter {
-    set python.branch   3.6
-}
-
-subport pypy37-tkinter {
-    set python.branch   3.7
+    set python.branch   3.11
+    version $pypy_versions(3.11)
+    github.tag_prefix   release-pypy3.11-v
 }
 
 subport pypy38-tkinter {
     set python.branch   3.8
+    version $pypy_versions(3.8)
+    github.tag_prefix   release-pypy3.8-v
 }
 
 subport pypy39-tkinter {
     set python.branch   3.9
+    version $pypy_versions(3.9)
+    github.tag_prefix   release-pypy3.9-v
+}
+
+subport pypy310-tkinter {
+    set python.branch   3.10
+    version $pypy_versions(3.10)
+    github.tag_prefix   release-pypy3.10-v
+}
+
+subport pypy311-tkinter {
+    set python.branch   3.11
+    version $pypy_versions(3.11)
+    github.tag_prefix   release-pypy3.11-v
 }
 
 if {$subport == ${name}} {
```

String comparisons must be done with `eq` not `==`:

```suggestion
if {${subport} eq ${name}} {
```